### PR TITLE
Bulq-1500 UPS signature tracking endpoint

### PR DESCRIPTION
**Pull request by kwoodard22**

### CODE REVIEW

**Comment by jec006** [comment]
@kwoodard22 this included #45 right?

**Comment by jec006** [comment]
@nevern02 can you look at this one instead actually

**Comment by kwoodard22** [comment]
@jec006 Yes, it includes https://github.com/optoro/active_shipping/pull/45. I moved the delivery info to the tracking response & included specs.

**Comment by nevern02** [comment]
Looks as good as anything in this gem can look.

File: lib/active_shipping/shipping/carriers/ups.rb [hunk 1]
```diff
@@ -506,18 +521,23 @@ def parse_tracking_response(response, options={})
               end
             end
             if shipment_events.last.name.downcase == 'delivered'
-              shipment_events[-1] = ShipmentEvent.new(shipment_events.last.type, shipment_events.last.name, shipment_events.last.time, destination)
+              if options[:signature_tracking]
+                signed_by = delivered_activity.get_text('ActivityLocation/SignedForByName').to_s
+                delivery_signature = delivered_activity.get_text('Document/Content').to_s
+              end
+              shipment_events[-1] = ShipmentEvent.new(shipment_events.last.type, shipment_events.last.name, shipment_events.last.time, destination, signed_by, signature)
```
Comment (jec006) [inline lib/active_shipping/shipping/carriers/ups.rb]: I think you need to remove signature and the like from here too

**Pull request merged**